Yoast/wordpress-seo
# Add czech researcher to plugin and app

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Adds Czech to the supported languages list

## Relevant technical choices:

* `cs` is used as the locale of Czech instead of `cz`. 

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

The steps below serve to smoke-test the Czech functionality.
* Test a post in Czech in following environment:

1. A normal post
2. Taxonomy page
3. A normal post in Elementor

Steps to test:

1.  Confirm that the SEO and readability analysis load
2. Set a keyphrase, add it to the text and see that it's recognized
3. Change some readability metrics (e.g., add transition words) and confirm that the analysis updates accordingly


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-695
